Frends.AmazonS3.DownloadObject/Frends.AmazonS3.DownloadObject/Definitions/Enums.cs

```diff
@@ -0,0 +1,73 @@
+﻿
+namespace Frends.AmazonS3.DownloadObject
+{
+    /// <summary>
+    /// AWS regions.
+    /// </summary>
+    public enum Region
+    {
+        #region regions
```

Comment:
No need for region here.

Reply:
It's easier for me to read the code when you can collapse pre-signed or AWS creds part out of sight in time to time. This doesn't affect the code.